central/debug/service/db_diagnostics.go

```diff
@@ -0,0 +1,82 @@
+package service
+
+import (
+	"context"
+	"os/exec"
+	"strings"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/stackrox/rox/central/globaldb"
+)
+
+const (
+	unableToGetConnectionInfo = "Unable to get the connection string"
+)
+
+func buildDBDiagnosticData(ctx context.Context, dbConfig *pgxpool.Config, dbPool *pgxpool.Pool) centralDBDiagnosticData {
+	diagnosticData := centralDBDiagnosticData{}
+
+	// Add the database version if Postgres
+	diagnosticData.Database = "PostgresDB"
+	diagnosticData.DatabaseServerVersion = globaldb.GetPostgresVersion(ctx, dbPool)
+	// Get client software version
+	diagnosticData.DatabaseClientVersion = getDBClientVersion()
+	// Get extensions
+	if dbConfig != nil {
+		diagnosticData.DatabaseConnectString = strings.TrimSpace(strings.Replace(dbConfig.ConnString(), dbConfig.ConnConfig.Password, "REDACTED", -1))
+	} else {
+		diagnosticData.DatabaseConnectString = unableToGetConnectionInfo
+	}
+	diagnosticData.DatabaseExtensions = getPostgresExtensions(ctx, dbPool)
+
+	return diagnosticData
+}
+
+func getDBClientVersion() string {
+	options := []string{
+		"-V",
+	}
+
+	clientVersion, err := exec.Command("pg_dump", options...).Output()
+	if err != nil {
+		log.Errorf("Unable to get client version:  %v", err)
```

Comment:
It could be more specific. `Unable to get Postgres client version` or `Unable to get pg_dump version`.